backend/Wheelingful.API/Extensions/Validation/BeValidBase64Extension.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Wheelingful.API.Extensions.Validation;
+
+public static class BeValidBase64Extension
+{
+    public static bool BeValidBase64(this string value)
+    {
+        try
+        {
+            _ = Convert.FromBase64String(value);
```

Comment:
there should be a method Convert.TryFromBase64String

Reply:
There is one, but that would be no better. It requires to write in variables I'm not going to use:
public static bool TryFromBase64String (string s, Span<byte> bytes, out int bytesWritten);